packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerBackButton.tsx

```diff
@@ -36,22 +36,19 @@ export const NavigationDrawerBackButton = ({
   title,
 }: NavigationDrawerBackButtonProps) => {
   const theme = useTheme();
-  const navigate = useNavigate();
   const navigationMemorizedUrl = useRecoilValue(navigationMemorizedUrlState);
 
   return (
     <StyledContainer>
-      <StyledIconAndButtonContainer
-        onClick={() => {
```

Comment:
I think the back button on record show pages was broken and this makes it more evident as it's now a 404

Reply:
Yes, please!

Reply:
Hey @FelixMalfait 
We've fixed this


https://github.com/twentyhq/twenty/assets/140154534/f90a83ef-cdb3-45fd-b740-0f3117521714


cc @lucasbordeau